File: leetcode/Arrays/Easy/move zeros/brute.cpp
#include <iostream>
#include<vector>
using namespace std;


class Solution{
public:
    void moveZeros(vector<int>& nums){
        int k = 0;
        for(int i = 0; i < nums.size(); i++){
            if(nums[i] != 0){
                nums[k] = nums[i];
                k++;
            }
        }
        for(int i = k; i < nums.size(); i++){
            nums[i] = 0;
        }
    }
};

int main() {
    Solution m;;
    vector<int> nums = {0, 1, 0, 3, 12};
    m.moveZeros(nums);
    for(int i = 0; i < nums.size(); i++){
        cout<<nums[i]<<endl;
    }
    return 0;
}

File: leetcode/Arrays/Easy/Find Pivot Index/brute.py
from typing import List

class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        total = sum(nums)
        left = 0

        for i in range(len(nums)):
            right = total - left - nums[i]

            if left == right:
                return i

            left = left + nums[i]

        return -1


# -------- Driver Code --------
nums = [1, 7, 3, 6, 5, 6]
obj = Solution()
result = obj.pivotIndex(nums)

print("Pivot Index:", result)
